File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/go/F12.P1.go
package main

import "fmt"

func fallback() int32 {
	// BEGIN PROBE F12.P1
	var o *int32 = nil
	var n int32
	if o != nil {
		n = *o
	}
	return n
	// END PROBE F12.P1
}

func main() {
	fmt.Println(fallback())
}

File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/java/F12.P1.java
import java.util.OptionalInt;

class Main {
    static int fallback() {
        // BEGIN PROBE F12.P1
        OptionalInt o = OptionalInt.empty();
        int n = o.orElse(0);
        return n;
        // END PROBE F12.P1
    }

    public static void main(String[] args) {
        System.out.println(fallback());
    }
}
